examples/aggregator/module/rust/src/lib.rs

```diff
@@ -92,7 +95,14 @@ impl AggregatorNode {
             bucket, svec
         );
 
-        match oak::grpc::client::Client::new("grpc-client", "").map(AggregatorClient) {
+        match oak::grpc::client::Client::new(&NodeConfiguration {
+            name: "grpc_client".to_string(),
+            config_type: Some(ConfigType::GrpcClientConfig(GrpcClientConfiguration {
+                address: "127.0.0.1:8888".to_string(),
+            })),
+        })
```

Comment:
Having helpers in the SDK for configuring standard pseudo-Node types would be helpful across all of the examples, e.g. so this could look something like:
```Rust
        match oak::grpc::client::Client::new(&oak::node_config::grpc_client("127.0.0.1:888"))
        .map(AggregatorClient)
        {
```

Reply:
Nice idea, done.